src/client/game/BulletPool.js

```diff
@@ -0,0 +1,29 @@
+var NUM_BULLETS = 10;
```

Comment:
I would think we would want the player to have unlimited ammo. Or is this the number of bullets that can be on the screen at a time?